.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
```

Comment:
I think the benefits are minimal/nonexistent and it's likely to slow down our build. I was just curious why/if we had to specify the xcode version. Makes sense now.